bot/tsconfig.json

```diff
@@ -1,43 +1,40 @@
 {
-    "compilerOptions": {
-        "target": "ES2018",
-        "module": "ESNext",
-        "moduleResolution": "Node",
-        "lib": [
-            "ESNext",
-            "ESNext.AsyncIterable",
-            "DOM"
-        ],
-        "esModuleInterop": true,
-        "allowJs": false,
-        "sourceMap": true,
-        "strict": true,
-        "noEmit": true,
-        "experimentalDecorators": true,
-        "forceConsistentCasingInFileNames": false,
-        "resolveJsonModule": true,
-        "baseUrl": ".",
-        "paths": {
-            "~/*": [
-                "./*"
-            ],
-            "@/*": [
-                "./*"
-            ],
-        },
-        "types": [
-            "@nuxt/types",
-            "@types/node"
-        ]
-    },
-    "exclude": [
-        "node_modules",
-        ".nuxt",
-        "dist"
+  "compilerOptions": {
+    "rootDir": "./src",
+    "outDir": "dist",
+    "composite": true,
+    "incremental": true,
+    "target": "es2017",
+    "moduleResolution": "node",
+    "module": "esnext",
+    "declaration": true,
+    "declarationMap": true,
+    "inlineSourceMap": true,
+    "esModuleInterop": true,
+    "resolveJsonModule": true,
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noFallthroughCasesInSwitch": true,
+    "traceResolution": false,
+    "listEmittedFiles": true,
+    "listFiles": false,
+    "pretty": true,
+    "lib": [
+      "es2019",
+      "dom"
     ],
-    "references": [
-        {
-            "path": "../core"
-        }
```

Comment:
This is important part, which might be related to the question above, see the tsconfig reference about this